src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/appsettings.json

```diff
@@ -11,6 +11,7 @@
     "Host": "localhost",
     "Port": 22,
     "Username": "demo",
-    "Password": "demo"
+    "Password": "demo",
+    "SshPrivateKeyPath": "C:\\some\\path\\key.ppk"
```

Comment:
We may prefer to use an environment variable instead of reading from disk by using this constructor. https://github.com/sshnet/SSH.NET/blob/bc99ada7da3f05f50d9379f2644941d91d5bf05a/src/Renci.SshNet/PrivateKeyFile.cs#L95